tokens/src/core/components/general/link.json

```diff
@@ -2,26 +2,27 @@
   "typography": {
     "link": {
       "decoration": {
-        "base": { "value": "none", "type": "textDecoration", "source": "$link-decoration" },
-        "hover": { "value": "underline", "type": "textDecoration", "source": "$link-hover-decoration" },
+        "$type": "textDecoration",
+        "base": { "source": "$link-decoration", "$value": "none" },
```

Comment:
[inform, docs site bug] The "Links" section on the typography foundations page is incorrectly showing `text-decoration: underline;` on "standalone links" (left: local; right: prod):

<img width="1899" alt="image" src="https://github.com/user-attachments/assets/52d6c4ff-dd18-44bb-9f21-67e9c2bcae93">

It appears this is due to the migration of the example tables to `DataTable`, where the `TextCell` is wrapping the cell content with a `p`, which according to the table description, "For links inside a `p`...", the standalone links are actually getting treated as inline links even without the explicit `.inline-link` class.

Reply:
[inform] Implemented in follow-up commit: [`6676868` (#3186)](https://github.com/openedx/paragon/pull/3186/commits/6676868e6c46d70d91f0bdc42b7c2848e3b0aa7c)